### PR TITLE
[14.0][MIG] agreement

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration

### CODE REVIEW

**Comment by sergio-teruel** [comment]
@newtratip please check out this PR https://github.com/OCA/contract/pull/397

**Comment by sergio-teruel** [comment]
Sorry the PR is this https://github.com/OCA/contract/pull/578

**Comment by newtratip** [comment]
@sergio-teruel, I saw this PR https://github.com/OCA/contract/pull/578 not plan to work anymore and then it will closed. Should I check out on this one or waiting another PR supersede https://github.com/OCA/contract/pull/578

**My question :** agreement is changed to base_contract, it's ok. and I am migrating [agreement_legal](https://github.com/OCA/contract/pull/623) module. Should this module still be name `agreement_legal` or another name ?

Thanks !

**Comment by sergio-teruel** [comment]
Hi @newtratip I not sure... I asked Alexis about PR (https://github.com/OCA/contract/pull/578) (https://github.com/OCA/contract/pull/578#issuecomment-772433286) status... and he will not continue working on it, so I'm thinking... Why do we go to change the name...??? 
I think that we continue with "agreement" name.

File: agreement/views/agreement.xml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+-->
+<odoo>
+
+
+<record id="agreement_form" model="ir.ui.view">
+    <field name="name">agreement.form</field>
+    <field name="model">agreement</field>
+    <field name="arch" type="xml">
+        <form string="Agreement">
+            <sheet>
```
Comment (sergio-teruel) [inline agreement/views/agreement.xml]: Please add 
`<div class="oe_button_box" name="button_box"/>` so other modules can add smartbuttons to main view

File: agreement/views/agreement_type.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        </field>
+    </record>
+
+    <!-- Agreement Type Form View -->
+    <record id="agreement_type_form_view" model="ir.ui.view">
+        <field name="name">Agreement Type Form</field>
+        <field name="model">agreement.type</field>
+        <field name="arch" type="xml">
+            <form string="Agreement Type">
+                <sheet>
```
Comment (sergio-teruel) [inline agreement/views/agreement_type.xml:20]: Please add 
`<div class="oe_button_box" name="button_box"/>` so other modules can add smartbuttons to main view

**Review by sergio-teruel — APPROVED** [review]
Little comment..

**Review by marcelsavegnago — APPROVED** [review]

File: agreement/views/agreement.xml (hunk 1 above)
Comment (newtratip) [reply agreement/views/agreement.xml]: done

File: agreement/views/agreement_type.xml (hunk 2 above)
Comment (newtratip) [reply agreement/views/agreement_type.xml:20]: done

**Comment by sergio-teruel** [comment]
Thanks @newtratip

File: agreement/security/agreement_security.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo noupdate="1">
+    <record id="agreement_rule" model="ir.rule">
+        <field name="name">Agreement multi-company</field>
+        <field name="model_id" ref="model_agreement" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```
Comment (sergio-teruel) [inline agreement/security/agreement_security.xml:12]: @newtratip Please change domain like as v 13.0 in my PR (https://github.com/OCA/contract/pull/628/files#diff-330aba9b038127959c7003d614d9c6985fba69f59eb728ac4594dd2872f5e812R12)
In v14.0 noupdate_changes.xml is not needed.

**Comment by newtratip** [comment]
Additional update, I check out this PR https://github.com/OCA/contract/pull/628

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: agreement/security/agreement_security.xml (hunk 3 above)
Comment (newtratip) [reply agreement/security/agreement_security.xml:12]: yes, changed domain Agreement multi-company rule and remove migrations folder.

**Review by sergio-teruel — APPROVED** [review]
Thanks!!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**